### PR TITLE
added tether messages

### PR DESCRIPTION
**Pull request by CraftyJax**
Messages for tether status

### CODE REVIEW

File: message_definitions/v1.0/planckAero.xml [hunk 1]
```diff
@@ -185,5 +203,17 @@
...
 	        <field type="uint8_t" name="fourhundredvolt_status">Status of the DECK 400v output. 0: Off 1: On</field>
         </message>
+        <message id="9012" name="PLANCK_DECK_TETHER_STATUS">
+           <description>Status of the TETHER system</description>
+           <field type="float" name="CABLE_OUT">Number of feet of cable spooled out</field>
+           <field type="float" name="CABLE_VELOCITY">Current cable speed in ft/s</field>
+           <field type="uint8_t" name="CABLE_TENSION">Current cable tension as percentag of max</field>
+           <field type="uint8_t" name="HV_POWER">Status of high voltage subsystem (see PLANCK_DECK_TETHER_HV_STATUS_TYPE)</field>
+           <field type="uint8_t" name="HV_TEMPERATURE">Current temp of  HV subsystem in def F</field>
+           <field type="uint8_t" name="HV_WATTS">Power being supplied to the tether in Watts</field>
```
Comment (almatthew) [inline message_definitions/v1.0/planckAero.xml]: 0->254 sufficient for describing watts?

File: message_definitions/v1.0/planckAero.xml [hunk 2]
```diff
@@ -185,5 +203,17 @@
 	        <field type="uint8_t" name="batt_status" enum="PLANCK_DECK_BATTERY_STATUS_TYPE">Status of the DECK battery. See PLANCK_DECK_BATTERY_STATUS_TYPE enum</field>
 	        <field type="uint8_t" name="fourhundredvolt_status">Status of the DECK 400v output. 0: Off 1: On</field>
         </message>
+        <message id="9012" name="PLANCK_DECK_TETHER_STATUS">
+           <description>Status of the TETHER system</description>
+           <field type="float" name="CABLE_OUT">Number of feet of cable spooled out</field>
+           <field type="float" name="CABLE_VELOCITY">Current cable speed in ft/s</field>
+           <field type="uint8_t" name="CABLE_TENSION">Current cable tension as percentag of max</field>
+           <field type="uint8_t" name="HV_POWER">Status of high voltage subsystem (see PLANCK_DECK_TETHER_HV_STATUS_TYPE)</field>
```
Comment (almatthew) [inline message_definitions/v1.0/planckAero.xml]: You can specify how this is described by adding an `enum="PLANCK_DECK_TETHER_HV_STATUS_TYPE"`

File: message_definitions/v1.0/planckAero.xml [hunk 3]
```diff
@@ -185,5 +203,17 @@
...
+           <field type="uint8_t" name="HV_TEMPERATURE">Current temp of  HV subsystem in def F</field>
```
Comment (almatthew) [inline message_definitions/v1.0/planckAero.xml]: Discrete degrees sufficient? (0->254)

File: message_definitions/v1.0/planckAero.xml [hunk 4]
```diff
@@ -185,5 +203,17 @@
...
+           <field type="uint8_t" name="HV_WATTS">Power being supplied to the tether in Watts</field>
+           <field type="uint8_t" name="SILENT_MODE">Status of silent mode (ON=1,OFF=0)</field>
+           <field type="uint8_t" name="SPOOL_STATUS">Current Spool Control mode (NORMAL=0,RETRIEVE=1,RELEASE=2,LOCKED=3,EOL_WARNING=4,EOL_ERROR=5)</field>
```
Comment (almatthew) [inline message_definitions/v1.0/planckAero.xml]: ditto on the enum description

**Review by almatthew — CHANGES_REQUESTED** [review]

File: message_definitions/v1.0/planckAero.xml (hunk 4 above)
Comment (CraftyJax) [reply message_definitions/v1.0/planckAero.xml]: done

File: message_definitions/v1.0/planckAero.xml (hunk 3 above)
Comment (CraftyJax) [reply message_definitions/v1.0/planckAero.xml]: Should be fixed now

**Review by almatthew — APPROVED** [review]

**Pull request merged**